client/scss/components/_dropdown.scss

```diff
@@ -1,5 +1,9 @@
 .w-dropdown__toggle {
   @apply w-px-2 w-text-text-label w-bg-transparent;
+  @media (prefers-contrast: more) {
+    @apply w-border w-border-solid w-border-text-label; 
+  }
+}
```

Comment:
This is currently throwing a syntax error due to the extraneous closing curly brace.

```suggestion
```

Also, could you add some screenshots to verify the issue you are fixing?